funcs_scripts.php

```diff
@@ -133,11 +184,38 @@ function info($message)
 
 /**
  * Output a warning message to the console
- * 
+ *
  * Example usage:
  * warning('This is something you might want to pay attention to')
  */
 function warning($message)
 {
     io()->warning($message);
 }
+
+/**
+ * Converts a cron expression to a human readable string
+ * Says UTC because that's what GitHub Actions uses
+ *
+ * Example usage:
+ * human_cron('5 4 * * 0')
+ * => 'At 4:05 AM UTC, only on Sunday'
+ */
+function human_cron(string $cron): string
+{
+    $str = (new ExpressionDescriptor($cron))->getDescription();
+    $str = preg_replace('#0([1-9]):#', '$1:', $str);
+    $str = preg_replace('# (AM|PM),#', ' $1 UTC,', $str);
+    return $str;
+}
+
+/**
+ * Creates a predicatable random int between 0 and $max based on the module name to be used with the % mod operator
+ */
+function predictable_random_int($max): int
+{
+    global $MODULE_DIR;
+    $chars = str_split($MODULE_DIR);
+    $codes = array_map(fn($c) => ord($c), $chars);
+    return (10000 + array_sum($codes)) % $max;
```

Comment:
```suggestion
    mt_srand(array_sum($codes));
    return mt_rand(0, $max);
```
May as well use build in random int functionality. Has the benefit of being clearer that it's producing a (predictably) random int within the limits without having to mentally walk through the math.

Reply:
It's not meant to be an actually random number, it's means to be "seemingly random though actually gives the exact same result each time"

Reply:
As discussed in person, seeding with `mt_srand()` and then running `mt_rand()` will always give you the same random value every time - and it has the stated advantage of being clearly what's happening and therefore we don't need a unit test for it.

Reply:
Updated